3.0.0-RC4/Dockerfile

```
@@ -0,0 +1,42 @@
+# Gatling is a highly capable load testing tool.
+#
+# Documentation: https://gatling.io/docs/2.3/
+# Cheat sheet: http://gatling.io/#/cheat-sheet/2.3
+
+FROM openjdk:8-jdk-alpine
+
+MAINTAINER Denis Vazhenin <denis.vazhenin@me.com>
+
+# working directory for gatling
+WORKDIR /opt
+
+# gating version
+ENV GATLING_VERSION 3.0.0-RC4
```

```suggestion
ENV GATLING_VERSION 3.0.0
```

Since the stable version was just released, maybe we can use the stable version instead?